src/parsed-feature-loading.ts

```diff
@@ -42,6 +42,7 @@ const parseDataTable = (astDataTable: any, astDataTableHeader?: any) => {
 
 const parseStepArgument = (astStep: any) => {
     if (astStep) {
+        // FIXME - astStep.argument no longer exists
```

Comment:
Step arguments have been removed somewhere in `@cucumber/gherkin` between "^17.0.0" --> "^24.1.0" ??

Reply:
Fixed.